### ISSUE TITLE
There was something wrong with Sophus when I tried to build SVO project

### ISSUE THREAD

**Issue by CurrySerena** [strasdat/Sophus#153]
test_depth_filter.cpp:(.text.startup+0x8f0): undefined reference to `Sophus::SE3::SE3(Eigen::Quaternion<double, 0> const&, Eigen::Matrix<double, 3, 1, 0, 3, 1> const&)'
test_depth_filter.cpp:(.text.startup+0x9b2): undefined reference to `Sophus::SE3::inverse() const'
test_depth_filter.cpp:(.text.startup+0x9d2): undefined reference to `Sophus::SE3::operator=(Sophus::SE3 const&)'
test_depth_filter.cpp:(.text.startup+0x11f8): undefined reference to `Sophus::SE3::inverse() const'
test_depth_filter.cpp:(.text.startup+0x1218): undefined reference to `Sophus::SE3::operator=(Sophus::SE3 const&)'
In short, there were a lot of errors about undefined reference to Sophus::SE3, so I was constructed by it, hope you can help me. I am looking forward to your reply.

**Comment by ruanhailiang** [comment]
hi，请问一下，您的问题解决了吗？如果方便的话，麻烦分享一下。我也碰到了同样的问题。

**Comment by strasdat** [comment]
this is about SVO using Sophus; which should be supported by the SVO project, not here.
